docs/ru/query_language/functions/array_functions.md

```diff
@@ -658,4 +658,115 @@ SELECT arrayReverseSort((x, y) -> -y, [4, 3, 5], [1, 2, 3]) AS res;
 
 Особенная функция. Смотрите раздел ["Функция arrayJoin"](array_join.md#functions_arrayjoin).
 
+## arrayDifference(arr)
+
+Принимает массив, возвращает массив разностей между парами соседних элементов. 
```

Comment:
Not descriptive enough. In what order the difference is applied?
I can read it as:
`[1 - 2, 3 - 4] = [-1, -1]`.

Reply:
Probably we should mind about the return type and the corner cases:

`SELECT arrayDifference([0, 10000000000000000000])`